substrate/frame/election-provider-support/src/traits.rs

```diff
@@ -127,23 +123,4 @@ where
 		voter_at: impl Fn(Self::VoterIndex) -> Option<A>,
 		target_at: impl Fn(Self::TargetIndex) -> Option<A>,
 	) -> Result<Vec<Assignment<A, Self::Accuracy>>, Error>;
-
-	/// Sort self by the means of the given function.
```

Comment:
Same, please undo the revert here. 100% harmless and just less merge aissues later.